examples/hello_ocean.jl

```diff
@@ -0,0 +1,35 @@
+# # Hello, ocean!
+#
+# > How inappropriate to call this planet Earth, when it is quite clearly _Ocean_.
+#
+#   --Arthur C. Clark
+
+using Oceananigans, Oceananigans.Grids, Plots
+
+grid = RegularCartesianGrid(size = (1, 64, 64),
+                            x = (0, 1), y = (-4, 4), z = (-4, 4),
+                            topology = (Periodic, Bounded, Bounded))
```

Comment:
also let's use `Flat` topology for the non-active dimension

Reply:
That NaNs right now, see #1024